### PR TITLE
Preparing release 0.47.10

### PR DESCRIPTION
**Pull request by carlosroman**
Releasing as `0.47.10` as only some minor changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+# 0.47.10 / 2023-08-07
+
+* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context and 'TLS' as min protocol version used in the `dummyTrustManager` [#436][]
```
Comment (scottopell) [inline CHANGELOG.md:5]: Maybe worth documenting the system property than can be used to override this. https://github.com/DataDog/jmxfetch/pull/436/files#diff-7b667f02364f0f4390a5f2db300223a30859c670c4a9515f94f40a2cad8fa4b6R29

`-Djmxfetch.min_tls_version=TLS` is the default, but can be overridden if needed. Legacy value is `SSL`, so if there are unexpected issues related to TLS, this system property can be used to override and go back to previous behavior.

Comment (urseberry) [inline CHANGELOG.md:5]:
```suggestion
* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context, and 'TLS' as min protocol version used in the `dummyTrustManager` [#436][] 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+# 0.47.10 / 2023-08-07
+
+* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context and 'TLS' as min protocol version used in the `dummyTrustManager` [#436][] 
+* [BUGFIX] Fixed issue race condition where an exception is thrown if the Agent hasn't finished initialising before JMXFetch starts to shut down [#449][]
+* [OTHER] Update management agent logic and comments for java 7 vs 8 vs 9 [#457][]
+
```
Comment (urseberry) [inline CHANGELOG.md:8]:
```suggestion
* [BUGFIX] Fixed issue race condition where an exception is thrown if the Agent hasn't finished initializing before JMXFetch starts to shut down [#449][]
* [OTHER] Update management agent logic and comments for Java 7 vs 8 vs 9 [#457][]

```

**Review by urseberry — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 Changelog
 =========
+# 0.47.10 / 2023-08-08
+
+* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context, and 'TLS' as min protocol version used in the `dummyTrustManager` (configurable using the flag `jmxfetch.min_tls_version`, e.g. `-Djmxfetch.min_tls_version=TL`) [#436][]
```
Comment (scottopell) [inline CHANGELOG.md:5]:
```suggestion
* [IMPROVEMENT] Improvements in how JMXFetch handles communicating back to the Agent. The TLS of the HTTP client used can now be configured, extra logging has been added around the SSL Context, and 'TLS' as min protocol version used in the `dummyTrustManager` (configurable using the flag `jmxfetch.min_tls_version`, e.g. `-Djmxfetch.min_tls_version=TLS`) [#436][] 
```

**Review by cmetz100 — APPROVED** [review]

**Pull request merged**